docs/table_design/Data_distribution.md

```diff
@@ -529,28 +529,54 @@ Buckets reflect how data files are actually organized in StarRocks.
 
 - How to set the number of buckets at table creation
 
-  - Method 1: automatically set the number of buckets
+  - Method 1: automatically set the number of buckets (recommended)
 
-    Since v2.5.7, StarRocks can automatically set the number of buckets based on machine resources and data volume for a partition.
+    - Table configured with hash bucketing
 
-    Example:
+      Since v2.5.7, StarRocks can automatically set the number of buckets based on machine resources and data volume for a partition at table creation.
+      > **NOTICE**
+      >
+      > If the raw data size of a partition exceeds 100 GB, we recommend that you manually configure the number of buckets using the Method 2.
+
+      Example:
+
+      ```sql
+      CREATE TABLE site_access (
+          site_id INT DEFAULT '10',
+          city_code SMALLINT,
+          user_name VARCHAR(32) DEFAULT '',
+          pv BIGINT SUM DEFAULT '0')
+      AGGREGATE KEY(site_id, city_code, user_name)
+      DISTRIBUTED BY HASH(site_id,city_code); -- Do not need to set the number of buckets
+      ```
+
+    - Table configured with random bucketing
+
+      Since v2.5.7, StarRocks can automatically set the number of buckets based on machine resources and data volume for a partition **at table creation**. Since v3.2.0, StarRocks further optimizes the logic for automatically setting the number of buckets. In addition to setting the number of buckets in a partition **at table creation**, StarRocks also **dynamically and accordingly increases** the number of buckets in a partition **during data loading**, based on the cluster capacity and the volume of loaded data. So it not only provides more convenience for users, but also does improve performance of loading bulk data.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/e23c2fa5-6cb8-49ce-9fd5-12bffae21246)
-> This not only makes table creation easier but also increases bulk load performance.